dotcom-rendering/src/web/server/pageTemplate.ts

```diff
@@ -300,10 +301,10 @@ https://workforus.theguardian.com/careers/product-engineering/
 				</script>
 
 				${bork ? `<script>(${fid.toString()})()</script>` : ''}
+				${bork ? `<script>(${fcp.toString()})();</script>` : ''}
```

Comment:
if you like it when things line up…

```suggestion
				${bork ? `<script>(${fcp.toString()})()</script>` : ''}
```